### PR TITLE
Change nginx.conf to match with pulp_installer one

### PR DESCRIPTION
**Pull request by fao89**
https://github.com/pulp/pulp_installer/blob/master/roles/pulp-webserver/templates/nginx.conf.j2
[noissue]

### CODE REVIEW

File: templates/travis/.travis/assets/nginx.conf [hunk 1]
```diff
@@ -41,17 +41,14 @@ http {
...
-
         location /pulp/content/ {
             proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
             proxy_set_header X-Forwarded-Proto $scheme;
             proxy_set_header Host $http_host;
             # we don't want nginx trying to do something clever with
             # redirects, we set the Host: header above already.
             proxy_redirect off;
-            proxy_pass http://127.0.0.1:24816;
+            proxy_pass http://pulp-content;
```
Comment (mdellweg) [inline templates/travis/.travis/assets/nginx.conf:51]: Where does this change come from? I am not sure it fits the single container architecture.

Comment (fao89) [reply templates/travis/.travis/assets/nginx.conf:51]: it comes from here:
https://github.com/pulp/plugin_template/blob/master/templates/travis/.travis/assets/nginx.conf#L25-L27
and this change was applied to some plugins:
- https://github.com/pulp/pulp_rpm/blob/master/.travis/assets/nginx.conf
- https://github.com/pulp/pulp_ansible/blob/master/.travis/assets/nginx.conf

Comment (mdellweg) [reply templates/travis/.travis/assets/nginx.conf:51]: Thank you for clarifying. I did no see this. And nginx config always seems to be good for a surprise.

**Comment by fao89** [comment]
@dkliban @daviddavis could you merge it? I could not due to travis issue

**Pull request merged**